acme/challenges.go

```diff
@@ -1,16 +1,16 @@
 package acme
 
 // Challenge is a string that identifies a particular type and version of ACME challenge.
-type Challenge string
+type ChallengeType string
```

Comment:
I like this rename, but please don't forget the godoc; this'll emit a lint warning as-is.